src/components/ProjectListItem/ProjectListItem.tsx

```diff
@@ -62,25 +68,25 @@ export const ProjectListItem: React.FC<ProjectListItemProps & ComponentProps<typ
                     [s.ProjectListItemIcons_editable]: editable,
                 })}
             >
-                {nullable(
-                    editable,
-                    () => (
-                        <ProjectSubscriptionButtons
-                            id={id}
-                            starred={starred}
-                            watching={watching}
-                            stargizersCounter={stargizers}
-                        />
-                    ),
+                {nullable(!editable, () => (
                     <>
                         {nullable(starred, () => (
                             <IconStarSolid size="s" />
                         ))}
                         {nullable(watching, () => (
                             <IconEyeOutline size="s" {...watch.attr} />
                         ))}
```

Comment:
I think we don't need this anymore

It should be `<ProjectSubscriptionButtons view='icons'  editable={false} />`